mesheryctl/internal/cli/root/mesh/deploy.go

```diff
@@ -0,0 +1,175 @@
+package mesh
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/manifoldco/promptui"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	deployCmd = &cobra.Command{
+		Use:   "deploy",
+		Short: "deploy a service mesh in the kubernetes cluster",
+		Args:  cobra.MinimumNArgs(1),
+		Long:  `deploy service mesh in the connected kubernetes cluster`,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			log.Infof("Verifying prerequisites...")
+			if len(args) < 1 {
+				return errors.New("no service mesh specified")
```

Comment:
Will we be showing them a list of available service meshes to provision based on the connected adapters?

Reply:
Covering this in upcoming PR.